src/routes/coupon_types.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: elvis
+ * Date: 2018/11/11
+ * Time: 17:58
+ */
+
+use \Psr\Http\Message\ServerRequestInterface as Request;
+use \Psr\Http\Message\ResponseInterface as Response;
+
+// Get All Coupon types
+$app->get('/api/coupon/type', function (Request $request, Response $response) {
```

Comment:
Should be consistent with the other one. either both use `/coupon/type` or `couponTypes`